frontend/src/pages/person/_itemId/index.vue

```diff
@@ -157,116 +124,96 @@ import { getBlurhash } from '@/utils/images';
-        recursive: true,
-        includeItemTypes: [BaseItemKind.Book],
-        fields: Object.values(ItemFields),
-        userId: remote.auth.currentUserId.value
-      })
-    ).data.Items;
+const series =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Series],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    const photos = (
-      await remote.sdk.newUserApi(getItemsApi).getItems({
-        personIds: [itemId],
-        sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-        sortOrder: [SortOrder.Descending],
-        recursive: true,
-        includeItemTypes: [BaseItemKind.Photo],
-        fields: Object.values(ItemFields),
-        userId: remote.auth.currentUserId.value
-      })
-    ).data.Items;
+const books =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Book],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    let activeTab = 4;
+const photos =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Photo],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    if (movies?.length) {
-      activeTab = 0;
-    } else if (series?.length) {
-      activeTab = 1;
-    } else if (books?.length) {
-      activeTab = 2;
-    } else if (photos?.length) {
-      activeTab = 3;
-    }
+const activeTab = ref(4);
 
-    return { activeTab, movies, series, books, photos, item };
-  },
-  computed: {
-    birthDate(): string | null {
-      return this.item.PremiereDate
-        ? dateFnsFormat(new Date(this.item.PremiereDate), 'PPP').value
-        : null;
-    },
-    deathDate(): string | null {
-      return this.item.EndDate
-        ? dateFnsFormat(new Date(this.item.EndDate), 'PPP').value
-        : null;
-    },
-    birthPlace(): string | null {
-      return this.item.ProductionLocations
-        ? this.item.ProductionLocations[0]
-        : null;
-    }
-  },
-  watch: {
-    item: {
-      handler(value: BaseItemDto): void {
-        this.$route.meta.title = value.Name || '';
+if (movies?.length) {
+  activeTab.value = 0;
+} else if (series?.length) {
+  activeTab.value = 1;
+} else if (books?.length) {
+  activeTab.value = 2;
+} else if (photos?.length) {
+  activeTab.value = 3;
+}
```

Comment:
Please, add a comment specifying this is for rendering the first tab that has content